test.js

```diff
@@ -0,0 +1,58 @@
+const modulus = require(".");
```

Comment:
id probably rename this to lib or something

Reply:
Yep i changed the name of the project at some point